packages/node_modules/@webex/plugin-meetings/internal-README.md

```diff
@@ -0,0 +1,145 @@
+
+## Contribute to the `plugin-meetings` branch
+
+We will work off of the `plugin-meetings` branch from `master`.
+
+### Set up remotes
+
+```bash
+git remote add origin git@github.com:YOUR-USERNAME/spark-js-sdk.git
+git remote add upstream git@github.com:webex/spark-js-sdk.git
+```
+
+### Track `plugin-meetings`
+
+```bash
+git checkout --track upstream/plugin-meetings
+```
+
+### Create a feature branch
+
+> You may find prefixing feature branches with the JIRA ticket number helpful, like in this example.
+
+```bash
+git checkout -b 39555-new-feature plugin-meetings
+```
+
+### Run the sample
+
+```bash
+npm run build
+PACKAGE="@webex/plugin-meetings" npm run serve:package
+```
+
+Visit https://localhost:8000/
+
+---
+
+🔮 Do your magical feature development here!
+
+---
+
+### Run tests
+
+Use `npm run distsrc` to point each package's `main` entry at the raw src and let `babel-node` compile on the fly.
+
+```bash
+# Point all package.json files to the src directory instead of the dist directory.
+npm run distsrc
+# Run the package tests.
+npm test -- --package @webex/plugin-meetings
+```
+
+You can use the `--unit`, `--integration`, `--automation`, and `--documentation` switches to control what types of tests you run and `--node` and `--browser` to control which environments your tests run in.
+
+`--browser --karma-debug` will run the browser tests with `{singleRun: false}`, thus allowing automatic rerunning everytime you save a file (though, karma does eventually get confused and you need to interrupt and restart the command).
+
+```bash
+# Example running browser tests in debug mode.
+npm test -- --package @ciscospark/plugin-teams --browser --karma-debug
+```
+
+> See [SCRIPTS.md](https://github.com/webex/spark-js-sdk/blob/master/SCRIPTS.md) to learn how to run tests on [SauceLabs](https://saucelabs.com/) and more.
+
+```bash
+# Point all package.json files back to the dist directory.
+npm run srcdist
+```
+
+### Commit changes
+
+The SDK enforces relatively strict commit messages to version its packages and keep language similar.
+
+Here is an example committing a new feature. Notice the *package name* in parentheses.
+
+```bash
+git commit -a -m 'feat(plugin-meetings): add ability to join a meeting'
+```
+
+See [Git Commit Guidelines](https://github.com/webex/spark-js-sdk/blob/plugin-meetings/CONTRIBUTING.md#git-commit-guidelines) for full details.
+
+### Submit a pull request
+
+---
+
+Prior to developing a new feature, be sure to search the [Pull Requests](https://github.com/webex/spark-js-sdk/pulls) for your idea to ensure you're not creating a duplicate change. Then, create a development branch in your forked repository for your idea and start coding!
+
+When you're ready to submit your change, first check that new commits haven't been made in the upstream's `master` branch. If there are new commits, rebase your development branch to ensure a fast-forward merge when your Pull Request is approved:
```

Comment:
PR should point to our tracked `upstream/plugin-meetings` branch

Reply:
Ah, I forgot to alter this stuff. Thanks!